libraries/kafka_config.rb

```diff
@@ -34,7 +34,8 @@ class KafkaConfig < Chef::Resource
                     'kafka.request.logger' => 'WARN',
                     'kafka.controller' => 'TRACE',
                     'kafka.log.LogCleaner' => 'INFO',
-                    'state.change.logger' => 'TRACE'
+                    'state.change.logger' => 'TRACE',
+                    'kafka.authorizer.logger' => 'WARN'
```

Comment:
I believe this logger is as of 0.9 of kafka.  Shouldn't it be a no-op if its an older version of kafka?